cogs5e/gametrack.py

```diff
@@ -466,11 +468,24 @@ async def customcounter(self, ctx, name=None, *, modifier=None):
         await character.commit(ctx)
 
         if new_value - counter.value:
-            out = f"{str(counter)}\n({abs(new_value - counter.value)} overflow)"
+            overflow = new_value - counter.value
+            changed = counter.value - old_value
+            out = f"{str(counter)}\n({abs(overflow)} overflow)"
+            # Calculations for Change (with overflow)
+            if not operator or operator == 'mod':
+                change = f' ({"+" if changed > 0 else ""}{changed})'
+            elif operator == 'set':
+                change = f' ({"+" if changed > 0 else ""}{changed})'
         else:
             out = str(counter)
-
-        result_embed.add_field(name=counter.name, value=out)
+            # Calculations for Change (without overflow)
+            if not operator or operator == 'mod':
+                change = f' ({"+" if modifier > 0 else ""}{modifier})'
```

Comment:
As a note, you can use `:+` in the f string to have it always show the sign:
`f' ({modifier:+})'` 

(Your way works, just thought I'd also point out an alternative you may not have been aware of)

Reply:
I did not know that, thanks, will update!